src/components/Breadcrumbs/BreadCrumbs.stories.tsx

```diff
@@ -1,20 +1,22 @@
 import { actions } from "@storybook/addon-actions";
+import { StoryObj } from "@storybook/react";
 import Breadcrumbs from ".";
 
 export default {
-  title: "Components/Breadcrumbs",
   component: Breadcrumbs,
```

Comment:
Might want to give this story a `title`, since the nesting seems unnecessary:

<img width="262" alt="Screenshot 2023-03-22 at 7 33 58 PM" src="https://user-images.githubusercontent.com/47064971/227062000-7fa7d457-63d3-4cca-8b4b-cfbba6c6dbaf.png">

Also, could we rename this file as `Breadcrumbs.stories.tsx`? (no capital C) :0